lib/stdlib/src/erl_lint.erl

```diff
@@ -3023,6 +3031,17 @@ is_valid_call(Call) ->
         _ -> true
     end.
 
+%% Raises a warning if we're remote-calling an unexported function (or
+%% referencing it with `fun M:F/A`), as this is likely to be unintentional.
+check_unexported_function(Anno, M, F, A, #lint{module=M}=St) ->
+    case (is_warn_enabled(unexported_function, St) andalso
+          (not gb_sets:is_element({F, A}, exports(St)))) of
```

Comment:
Nitpick - I don't see any reason for the extra parentheses. In my opinion, they decrease readability. Also, it is a style-break from most of the other code in this module.

```suggestion
    case is_warn_enabled(unexported_function, St) andalso
          not gb_sets:is_element({F, A}, exports(St)) of
```

Reply:
When emacs-formatted, the `not` and the `true` are placed on the same line without the parentheses, so with multi-line case expressions I have the habit of wrapping them in parentheses. Other multi-line cases in the module (however not all) follow this pattern.

Thanks for pointing it out though, I had done it manually and forgotten that the `andalso` must also be on the next line for the formatter to be happy.